File: Daily_Practice/fabonacci.py
# def fabonacci(num):
#     n1, n2 = 0, 1
#     if num <= 0:
#         print("Please enter a positive number...")
#     elif num == 1:
#         print(0)
#     else:
#         print("Fabonacci Sequence: ", end="")
#         for i in range(0,num):
#             print(n1, end=", ")
#             nth = n1 + n2
#             n1 = n2
#             n2 = nth

# fabonacci(10)


n1, n2 = 0, 1
for i in range(10):
    print(n1)
    nth = n1+n2
    n1 = n2
    n2 = nth 

File: Daily_Practice/Go_Debug_Practice/Fabonacci.go
package main

import "fmt"

func Fabonacci(num int) {
	var n1, n2 = 0, 1
	var nth int
	if num <= 0 {
		fmt.Println("Please enter a positive number...")
	} else if num == 1 {
		fmt.Println(n1)
	} else if num > 1 {
		fmt.Print("Fabonacci Sequence:", " ")
		for i := 0; i < num; i++ {
			fmt.Print(n1, ", ")
			nth = n1 + n2
			n1 = n2
			n2 = nth
		}
	}
}

func main() {
	Fabonacci(10)
}
